spec/index.html

```diff
@@ -298,6 +298,13 @@ <h3>rdf:Property</h3>
     <p><code>rdf:Property</code> is the class of RDF properties.
       <code>rdf:Property</code> is an instance of <a href="#ch_class"><code>rdfs:Class</code></a>.</p>
   </section>
+
+  <section id="ch_proposition">
+    <h3>rdf:Proposition</h3>
+    <p><code>rdf:Proposition</code> is the class of reified triples.
```

Comment:
Update this to the definition in the RDF representation?

Reply:
There is a PR for RDF Representation, I will handle this in a couple of days.